.github/workflows/linters.yml

```diff
@@ -6,43 +6,57 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  lighthouse:
-    name: Lighthouse
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.7.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
+  # lighthouse:
+  #   name: Lighthouse
+  #   runs-on: ubuntu-18.04
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-node@v1
+  #       with:
+  #         node-version: "12.x"
+  #     - name: Setup Lighthouse
+  #       run: npm install -g @lhci/cli@0.7.x
+  #     - name: Lighthouse Report
+  #       run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  # webhint:
+  #   name: Webhint
+  #   runs-on: ubuntu-18.04
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-node@v1
+  #       with:
+  #         node-version: "12.x"
+  #     - name: Setup Webhint
+  #       run: |
+  #         npm install --save-dev hint@6.x
+  #         [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
+  #     - name: Webhint Report
+  #       run: npx hint .
```

Comment:
- [ ] Let's make sure that the html&css + javascript linters are set up correctly. Follow [this link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and copy [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) to your `linters.yml` file. Make sure `webhint` and `lighthouse` configurations are added to the `linters.yml` file.